cmd/go-cache-plugin/setup.go

```diff
@@ -52,10 +52,15 @@ func initCacheServer(env *command.Env) (*gocache.Server, *s3util.Client, error)
 		return nil, nil, fmt.Errorf("create local cache: %w", err)
 	}
 
-	cfg, err := config.LoadDefaultConfig(env.Context(),
+	opts := []func(*config.LoadOptions) error{
```

Comment:
Shall we use the declared type here?
```suggestion
	opts := []config.LoadOptionsFunc{
```

Reply:
Thats unfortunately not possible, since the `config.LoadDefaultConfig` function expects the options not to be in this type but in the anonymous function style. Quite weird, but it's nothing we can control.

Reply:
Ah, right, well that's fine then.